pyflagser/flagser.py

```diff
@@ -231,7 +231,7 @@ def flagser_weighted(adjacency_matrix, max_edge_weight=None, min_dimension=0,
 
     # Create dictionary of return values
     out = dict()
-    out['dgms'] = [homology[0].get_persistence_diagram()[i]
+    out['dgms'] = [np.asarray(homology[0].get_persistence_diagram()[i])
```

Comment:
Suggestion for lines 228 onwards:
```
    # Call flagser binding
    homology = compute_homology(vertices, edges, min_dimension, _max_dimension,
                                directed, coeff, _approximation, filtration)[0]

    # Create dictionary of return values
    out = {
        'dgms': [np.asarray(d) for d in homology.get_persistence_diagram()],
        'betti': homology.get_betti_numbers(),
        'cell_count': homology.get_cell_count(),
        'euler': homology.get_euler_characteristic()
    }

```

Reply:
Nicely done, I also added that for unweighted counterpart.